.github/workflows/docs.yml

```
@@ -50,46 +48,30 @@ jobs:
             && pip3 install .
             && pip3 install openfermionpyscf
             && pip3 install aiohttp fsspec h5py
-            && pip freeze | grep -v 'file:///' > .github/stable/doc.txt.tmp
         build-command: "sphinx-build -b html . _build -W --keep-going"
 
-    - name: Prepare local repo
-      if: github.event.pull_request.head.repo.full_name == 'PennyLaneAI/pennylane'
-      run: |
-        git fetch
-        git config user.name "GitHub Actions Bot"
-        git config user.email "<>"
-        if git ls-remote --exit-code origin "refs/heads/${{ env.DEPS_BRANCH }}"; then
-          git checkout "${{ env.DEPS_BRANCH }}"
-        else
-          git checkout master
-          git pull
-          git checkout -b "${{ env.DEPS_BRANCH }}"
-        fi
-        mv -f .github/stable/doc.txt.tmp .github/stable/doc.txt
+    - name: Freeze dependencies
+      shell: bash
+      run: pip freeze | grep -v 'file:///' > doc.txt
```

```suggestion
      run: |
        pip freeze | grep -v 'file:///' > doc.txt
        cat doc.txt
```
Let's print it just so we know